web/frontend/src/components/utils/useChangeAction.tsx

```diff
@@ -155,6 +170,17 @@ const useChangeAction = (
 
   const getAction = () => {
     switch (status) {
+      case STATUS.INITIAL:
+        return (
+          <span>
+            <button id="close-button" className="election-btn" onClick={handleOpen}>
+              {t('open')}
```

Comment:
The `open` key is missing in the `en.json` file.

```ts
...
"open": "Open",
...

Reply:
Nice catch!